javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,91 @@
+function getFullname(firstname, surname, useFormalName){
+    
+    let fullname = (useFormalName == true) ? ("Lord" + "  "+firstname +"  "+ surname) :(firstname +"  "+ surname);
+
+    console.log(fullname);
+
+}
```

Comment:
Love the use of the ternary here.
Just point something out here, your functions should only perform one single task. In this case, `getFullName`, must only get the full name and return it, not also print it. Printing should be the responsibility of some other function.